src/Copy/Keys.elm

```diff
@@ -68,38 +68,32 @@ type Key
     | DefinitionConciseP1
     | DefinitionConciseP2
     | DefinitionConciseP3
-    | DefinitionMoreLink
+    | SplitterAffirmation
```

Comment:
Is this still a splitter affirmation?

Reply:
I'm hopnig it will move back down, right now it's just affirmation